VerilogCodingStyle.md

```diff
@@ -523,14 +528,33 @@ that the identical parts are directly above one another. This alignment makes it
 easy to see which characters are the same and which characters are different
 between lines.
 
+Each block of code, separated by an empty line, is treated as separate "table".
+
 Use spaces, not tabs.
 
 For example:
 
+:+1:
 ```systemverilog
 logic [7:0]  my_interface_data;
 logic [15:0] my_interface_address;
 logic        my_interface_enable;
+
+logic       another_signal;
+logic [7:0] something_else;
+```
+
+:+1:
+```systemverilog
+mod u_mod (
+  .clk_i,
+  .rst_ni,
+  .sig_i (my_signal_in),
+  .sig2_i(my_signal_out),
+
+  .in_another_block_i(my_signal_in),
+  .sig3_i            (something)
+);
```

Comment:
Would it be worth an additional example with a comment rather than a blank line?

```
mod u_mod (
  .clk_i,
  .rst_ni,
  .sig_i          (my_signal_in),
  .sig2_i         (my_signal_out),
  // comment with no blank line maintains the block
  .in_same_block_i(my_signal_in),
  .sig3_i         (something)
);